test/chains/agreement.test.ts

```diff
@@ -39,27 +24,7 @@ describe('Agreement', () => {
     await query('manifests').delete('*')
   })
 
-  test('Does not calculate score without master_key', async () => {
```

Comment:
This test was removed because it wasn't really valid. The VHS was originally using `manifests` to determine master keys, but that's provided in validations as well, so it's easier to pull master keys from the validations themselves. This test originally checked that agreement scores wouldn't generate if there weren't values in `manifests`.